File: video/users/client-initialization-rooms/client-initialization-rooms.m
// Join a room 
TVIConnectOptions *connectOptions = [TVIConnectOptions optionsWithToken:self.accessToken
                                                                  block:^(TVIConnectOptionsBuilder * _Nonnull builder) {
  builder.roomName = @"my-room";
}];

TVIRoom *room = [TwilioVideo connectWithOptions:connectOptions delegate:self];

File: video/users/client-initialization-rooms/client-initialization-rooms.java
// Connect to a room
ConnectOptions connectOptions = new ConnectOptions.Builder(accessToken)
                .roomName("my-room")
                .audioTracks(localAudioTracks)
                .videoTracks(localVideoTracks)
                .build();
Room room = Video.connect(context, connectOptions, roomListener());
